model/task/results.go

```diff
@@ -69,7 +71,7 @@ func GetResultCounts(tasks []Task) *ResultCounts {
 
 	for _, t := range tasks {
 		out.Total++
```

Comment:
GetDisplayStatus() does not sum up inactive, and since this is used in a rest route, I will leave it as ResultStatus in case someone relies on that value.